examples/configuration/UsingSSHToConfigureNameResolution.ipynb

```diff
@@ -96,6 +147,23 @@
     "echo NODE_RESERVATION $NODE_RESERVATION"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Verify if lease has been created.\n",
```

Comment:
https://github.com/ChameleonCloud/jupyterhub-deploy/blob/master/singleuser/bashrc.d/chameleon_functions.sh#L104

Reply:
Let's do it in a future version; we ran out of time here.